db/seeds.rb

```diff
@@ -0,0 +1,157 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+
+require 'json'
```

Comment:
Do we need to require if the gem is installed?

Reply:
not sure! i will have a search. good thought though!